app/grandchallenge/evaluation/models.py

```diff
@@ -509,6 +510,14 @@ class Phase(FieldChangeMixin, HangingProtocolMixin, UUIDModel):
         default=0,
         help_text="The total compute cost for this phase in Euro Cents, including Tax",
     )
+    parent = models.ForeignKey(
+        "self",
+        on_delete=models.PROTECT,
+        related_name="children",
+        null=True,
+        blank=True,
+        help_text="Is this phase dependent on another phase? If selected, submissions to the current phase will only be possible after a succesful submission has been made to the parent phase.",
```

Comment:
We should probably add a warning about enabling this. On deadline day if you require a successful submission to a sanity check phase in order to submit to the test phase it could prevent people submitting if the sanity check takes a long time to execute. Don't know how to phrase this better!

Reply:
Good idea. I also struggle to find a different wording. I think this is as clear as it can get. People will just need to stand still and think for a moment when enabling this.